src/Compilers/VisualBasic/Portable/Parser/ParseExpression.vb

```diff
@@ -1220,21 +1220,76 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
             Return ReportSyntaxError(SyntaxFactory.XmlName(Nothing, SyntaxFactory.XmlNameToken("", SyntaxKind.XmlNameToken, Nothing, Nothing)), ERRID.ERR_ExpectedXmlName)
         End Function
 
-        ' File: Parser.cpp
-        ' Lines: 1501 - 1501
-        ' Expression* .Parser::ParseParenthesizedExpression( [ _Inout_ bool& ErrorInConstruct ] )
-        Private Function ParseParenthesizedExpression() As ParenthesizedExpressionSyntax
+        Private Function ParseParenthesizedExpressionOrTupleLiteral() As ExpressionSyntax
             Debug.Assert(CurrentToken.Kind = SyntaxKind.OpenParenToken)
 
-            ' "(" expr ")"
+            ' "(" expr ")"              'parethesized
+            ' "(" Name:= ....           'parse a tuple
+            ' "(" expr, ....            'parse a tuple
+
             Dim openParen As PunctuationSyntax = Nothing
             TryGetTokenAndEatNewLine(SyntaxKind.OpenParenToken, openParen)
-            Dim Operand = ParseExpressionCore()
+
+            If (CurrentToken.Kind = SyntaxKind.IdentifierToken AndAlso
+                PeekToken(1).Kind = SyntaxKind.ColonEqualsToken) Then
+
+                Dim argumentName = ParseIdentifierNameAllowingKeyword()
+                Dim colonEquals As PunctuationSyntax = Nothing
+                TryGetTokenAndEatNewLine(SyntaxKind.ColonEqualsToken, colonEquals)
+
+                Dim nameColonEquals = SyntaxFactory.NameColonEquals(argumentName, colonEquals)
+                Dim firstArgument = SyntaxFactory.SimpleArgument(nameColonEquals, ParseExpressionCore())
+
+                Return ParseTheRestOfTupleLiteral(openParen, firstArgument)
+            End If
+
+            Dim operand = ParseExpressionCore()
+
+            If (CurrentToken.Kind = SyntaxKind.CommaToken) Then
+                Dim firstArgument = SyntaxFactory.SimpleArgument(nameColonEquals:=Nothing, expression:=operand)
+
+                Return ParseTheRestOfTupleLiteral(openParen, firstArgument)
+            End If
 
             Dim closeParen As PunctuationSyntax = Nothing
             TryEatNewLineAndGetToken(SyntaxKind.CloseParenToken, closeParen, createIfMissing:=True)
 
-            Return SyntaxFactory.ParenthesizedExpression(openParen, Operand, closeParen)
+            Return SyntaxFactory.ParenthesizedExpression(openParen, operand, closeParen)
+        End Function
+
+        Private Function ParseTheRestOfTupleLiteral(openParen As PunctuationSyntax, firstArgument As SimpleArgumentSyntax) As TupleExpressionSyntax
```

Comment:
This will create a single-element tuple from the syntax (name := 1), which I think is a good thing. We need to make sure that is handled well by the rest of the compiler.